_pages/our-approach/keep-refining.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Keep refining
+---
+
+Content design is an ongoing process, and even published content isn’t really “done,” in a traditional sense — it’s not a static entity. To ensure that your content is helping users, you need to keep refining your content over time.
+
```

Comment:
@nicoleslaw Hey! I just took another look at the intro and think it might be nice to add a second paragraph here. What do you think of this?

`As is the case when you’re creating content, it’s a best practice to base your refinements on insights from your users. This section addresses ways to test your content’s effectiveness and includes tips for archiving and deleting content without disrupting your users’ experience.`

(Feel free to add/delete/edit as you like!)